cmd/commands/runtime.go

```diff
@@ -301,28 +303,13 @@ func runtimeInstallCommandPreRunHandler(cmd *cobra.Command, opts *RuntimeInstall
 		return err
 	}
 
-	err = askUserIfToInstallDemoResources(cmd, &opts.InstallDemoResources)
-	handleCliStep(reporter.InstallStepPreCheckShouldInstallDemoResources, "Asking user is demo resources should be installed", err, false)
-	if err != nil {
-		return err
-	}
+	askUserIfToInstallDemoResources(cmd, &opts.InstallDemoResources)
 
-	if opts.GsCloneOpts.Auth.Username == "" {
-		opts.GsCloneOpts.Auth.Username = opts.InsCloneOpts.Auth.Username
-	}
-
-	if opts.GsCloneOpts.Auth.Password == "" {
-		opts.GsCloneOpts.Auth.Password = opts.InsCloneOpts.Auth.Password
-	}
-
-	if opts.GsCloneOpts.Repo == "" {
-		host, orgRepo, _, _, _, suffix, _ := aputil.ParseGitUrl(opts.InsCloneOpts.Repo)
-		opts.GsCloneOpts.Repo = host + orgRepo + "_git-source" + suffix + "/resources" + "_" + opts.RuntimeName
-	}
-
-	if opts.GsCloneOpts.Provider == "" {
-		opts.GsCloneOpts.Provider = opts.InsCloneOpts.Provider
-	}
+	opts.GsCloneOpts.Provider = opts.InsCloneOpts.Provider
+	opts.GsCloneOpts.Auth = opts.InsCloneOpts.Auth
+	opts.GsCloneOpts.Progress = opts.InsCloneOpts.Progress
+	host, orgRepo, _, _, _, suffix, _ := aputil.ParseGitUrl(opts.InsCloneOpts.Repo)
+	opts.GsCloneOpts.Repo = host + orgRepo + "_git-source" + suffix + "/resources" + "_" + opts.RuntimeName
```

Comment:
but it's not a check

Reply:
sorry, i meant in the `runtimeInstallCommandPreRunHandler`
so that once the actual `RunRuntimeInstall` func is running, all the flags and options will already have their values set.

Reply:
my bad. i should have looked at it more closely. sorry